cmd/secretkeepassxc.go

```diff
@@ -127,10 +128,43 @@ func (c *Config) keePassXCAttributeFunc(entry, attribute string) string {
 	return outputStr
 }
 
+func readPassword(prompt string) (pw []byte, err error) {
+	fd := int(os.Stdin.Fd())
+	if terminal.IsTerminal(fd) {
+		fmt.Fprint(os.Stderr, prompt)
+		pw, err = terminal.ReadPassword(fd)
+		fmt.Fprintln(os.Stderr)
+		return
+	}
+
+	var b [1]byte
```

Comment:
Note that this branch does not print the prompt, is that deliberate?

Also, I'm a bit surprised that `terminal.ReadPassword` doesn't work on Windows. Are you sure that's the case?

Reply:
Ah, I've just seen [your comment in the other issue explaining this](https://github.com/twpayne/chezmoi/issues/768#issuecomment-634388601), thanks.